sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -209,7 +209,7 @@ We guarantee that all client instance methods are thread-safe and independent of
 The Azure.Security.KeyVault.Keys package supports synchronous APIs.
 
 The following section provides several code snippets using the `client` [created above](#create-keyclient), covering some of the most common Azure Key Vault key service related tasks:
-
+<!--
```

Comment:
Why are we commenting all these out?

Reply:
the links here don't work in the rendered HTML

Reply:
Instead of commenting, fix the links please.
Links are currently missing.  Follow what we do on `sdk/core/azure-core/README.md`, adding the links in the bottom and reference here.

For example, for `[Create a key]` , we should link `https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/keyvault/azure-security-keyvault-keys/samples/sample1_hello_world.md#creating-a-key`.

After updating the links, we could get rid of the sections below, as they are duplicated with each sample md file